include/smack/Prelude.h

```diff
@@ -0,0 +1,25 @@
+//
+// This file is distributed under the MIT License. See LICENSE for details.
+//
+
+#ifndef PRELUDE_H
+#define PRELUDE_H
+
+#include "smack/BoogieAst.h"
```

Comment:
Looks like we can avoid this include here, and include in Prelude.cpp instead, since this header does not use BoogieAst directly. Otherwise modules which include the Prelude header would have to be recompiled every time you changed the BoogieAst header.